error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/package-info.java

```diff
@@ -1,4 +1,4 @@
 /** Picnic Error Prone Contrib checks. */
+@org.jspecify.nullness.NullMarked
```

Comment:
Swapping order of the annotations for sorting and consistency with the other file.